ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.25.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  base:
+    name: "4.11"
```

Comment:
hm , I would expect this to look more like https://github.com/openshift/release/blob/master/ci-operator/config/cri-o/cri-o/cri-o-cri-o-main__ci.yaml than https://github.com/openshift/release/blob/master/ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml

Reply:
Yeah, this makes more sense to me.